src/pages/CreateProfile/BasicInfo.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
```

Comment:
Should: resolve errors in console
Getting the following error in the console from this page:

Warning: Failed prop type: The prop `xs` of `Grid` must be used on `item`.

Reply:
I'm not getting this error in the console so not sure how to fix

Reply:
should be fixed